app-editors/zoinks/zoinks-0.4.1-r1.ebuild

```diff
@@ -1,17 +1,17 @@
-# Copyright 1999-2014 Gentoo Foundation
+# Copyright 1999-2016 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 # $Id$
 
-EAPI=5
-inherit eutils toolchain-funcs
+EAPI=6
+inherit eutils
 
 DESCRIPTION="programmer's text editor and development environment"
 HOMEPAGE="http://zoinks.mikelockwood.com"
 SRC_URI="http://${PN}.mikelockwood.com/download/${P}.tar.gz"
 
 LICENSE="GPL-2"
 SLOT="0"
-KEYWORDS="amd64 ppc x86"
+KEYWORDS="~amd64 ~ppc ~x86"
```

Comment:
You're still bumping an ebuild in place. Your edits are ok, just don't kill the old `zoinks-0.4.1.ebuild`